topeft/modules/corrections.py

```diff
@@ -1168,33 +1168,7 @@ def AttachElectronSF(electrons, year, looseWP=None):
     reco_up = ak.unflatten(reco_up_flat, ak.num(pt))
     reco_do = ak.unflatten(reco_do_flat, ak.num(pt))
 
-    if is_run3:
-        loose_sf_flat = None
-        loose_up_flat = None
-        loose_do_flat = None
-        pt_mask = ak.flatten((pt >= 10))
-        egm_args = [eta_flat, pt_bin_flat]
-        if "2023" in year:
-            egm_args.append(phi_flat)
-        loose_sf_flat = ak.where(
-            ~pt_mask,
-            1,
-            ceval[egm_tag].evaluate(egm_year, "sf", looseWP, *egm_args)
-        )
-        loose_up_flat = ak.where(
-            ~pt_mask,
-            1,
-            ceval[egm_tag].evaluate(egm_year, "sfup", looseWP, *egm_args)
-        )
-        loose_do_flat = ak.where(
-            ~pt_mask,
-            1,
-            ceval[egm_tag].evaluate(egm_year, "sfdown", looseWP, *egm_args)
-        )
-        loose_sf = ak.unflatten(loose_sf_flat, ak.num(pt))
-        loose_up = ak.unflatten(loose_up_flat, ak.num(pt))
-        loose_do = ak.unflatten(loose_do_flat, ak.num(pt))
```

Comment:
Is this code we need to keep?

Reply:
@bryates if we don't plan to use any loose flag from EGM POG (if I remember correctly what @sscruz reported in the previous PR), I am fine with removing this part for the Run 3 analysis

Reply:
I wasn't sure if setting the loose flag to `None` was a temporary thing until loose SFs are ready. If we don't expect to have loose SFs for Run 3, then we can remove these lines. Otherwise, we should put them back in and maybe raise an exception until they're ready.

Reply:
Got it. So, according to this comment from @sscruz :
https://github.com/TopEFT/topeft/pull/445#discussion_r1867310971

I would say, we can add to the original if statement the condition:
```
looseWP is not None
```
and it should properly handle the scenario where this if section is not deleted, but not actually used until we decide to implement any SFs for the EGM POG MVA, if we will eventually use it. What do you think?

Reply:
Sure, that works for me

Reply:
@sscruz let me know if you want me to apply this modification